tests/functional/amdModuleCircular.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+	<head lang="en">
+	<meta charset="UTF-8">
+	<title>AMD Loading Test</title>
+</head>
+
+<body>
+	<script src="../../loader.js"></script>
```

Comment:
In each of your HTML files, could you remove the indent on these script tags to end up with something like this:
```html
<body>
<script src="..."></script>
<script>
require.config(...);
require(...);
</script>
</body>
```